_posts/2020-08-31-community-update.md

```diff
@@ -0,0 +1,222 @@
+        <p> The first discussion to branch out of this "umbrella" FLIP is around support for a batch execution mode in the DataStream API (<a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-134%3A+Semantics+of+Bounded+Applications+on+the+DataStream+API">FLIP-134</a>).</p>
+      </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-135+Approximate+Task-Local+Recovery">FLIP-135</a></td>
+        <td><ul>
+        <li><b>Approximate Task-Local Recovery</b></li>
+        <p>To better accommodate recovery scenarios where a certain amount of data loss is tolerable, but a full pipeline restart is not desirable, the community plans to introduce a new failover strategy that allows to restart only the failed task(s). Approximate task-local recovery will allow users to trade consistency for fast failure recovery, which is handy for use cases like online training.</p>
+      </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-136%3A++Improve+interoperability+between+DataStream+and+Table+API">FLIP-136</a></td>
+        <td><ul>
+        <li><b>Improve the interoperability between DataStream and Table API</b></li>
+        <p>The Table API has seen a great deal of refactoring and new features in recent releases, but the interfaces to and from the DataStream API haven't been updated accordingly. The work in this FLIP will cover multiple known gaps to improve interoperability and expose important functionality also to the DataStream API (e.g. changelog handling).</p>
+      </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-139%3A+General+Python+User-Defined+Aggregate+Function+Support+on+Table+API">FLIP-139</a></td>
+        <td><ul>
+        <li><b>Support Stateful Python UDFs</b></li>
+        <p>Python UDFs have been supported in PyFlink <a href="https://flink.apache.org/news/2020/02/11/release-1.10.0.html#pyflink-support-for-native-user-defined-functions-udfs">since 1.10</a>, but were so far limited to stateless functions. The community is now looking to introduce stateful aggregate functions (UDAFs) in the Python Table API.</p>
+        <p><b>Note: </b>Pandas UDAFs are covered in a separate proposal (<a href="https://cwiki.apache.org/confluence/display/FLINK/FLIP-137%3A+Support+Pandas+UDAF+in+PyFlink">FLIP-137</a>).</p>
+      </ul>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+For a complete overview of the development threads coming up in the project, check the [Flink Wiki](https://cwiki.apache.org/confluence/display/FLINK/Flink+Improvement+Proposals) and follow the feature discussions in the [@dev mailing list](https://lists.apache.org/list.html?dev@flink.apache.org).
```

Comment:
You could also link to the list of 1.12 features in the wiki here: https://cwiki.apache.org/confluence/display/FLINK/1.12+Release